packages/odata-common/src/request/odata-request-config.ts

```diff
@@ -20,7 +20,7 @@ const logger = createLogger({
  */
 export abstract class ODataRequestConfig {
   payload: Record<string, any> | string;
-  customServicePath: string;
+  basePath: string;
```

Comment:
[q] Should we ceep the custom prefix?

Reply:
I was wondering the same thing, however in the ticket one of the TODOs is:
`Rename the setCustomServicePath parameter for OData and replace it with setBasePath`
Therefore I assumed the removal of the `custom` would be applied here as well.